pkg/controller/jwks/jwks.go

```diff
@@ -130,12 +140,7 @@ func (c *Controller) getRealm(realmStr string) (*database.Realm, error) {
 	if err != nil {
 		return nil, errBadRealm
 	}
-	var realm *database.Realm
```

Comment:
The only way this returns errBadRealm now is if the string fails to parse as an int. But `FindRealm` shouldn't require an int. If it does, we should make that an interface and remove this function entirely and call c.db.FindRealm(..) in the main function.

Reply:
even easier!